data/instrumentation.yaml

```diff
@@ -47,11 +47,11 @@ languages:
       traces: stable
       metrics: not yet implemented
       logs: not yet implemented
-  ruby:
-    name: Ruby
+  js:
+    name: JavaScript
     status:
       traces: stable
-      metrics: not yet implemented
+      metrics: Release Candidate
       logs: not yet implemented
```

Comment:
The original status for logs was "Development", should we be preserving it here -- instead of "not yet implemented"?